scripts/deploy-rpc-playbooks.sh

```diff
@@ -29,20 +29,60 @@ source ${BASE_DIR}/scripts/functions.sh
 cd ${RPCD_DIR}/playbooks/
 
 # configure everything for RPC support access
-run_ansible rpc-support.yml
-
-# deploy and configure RAX MaaS
-if [[ "${DEPLOY_MAAS}" == "yes" ]]; then
-  run_ansible setup-maas.yml
+if [[ "${DEPLOY_SUPPORT_ROLE}" == "yes" ]]; then
+    run_ansible rpc-support.yml
 fi
 
 # deploy and configure the ELK stack
 if [[ "${DEPLOY_ELK}" == "yes" ]]; then
-  run_ansible setup-logging.yml
+    run_ansible setup-logging.yml
 fi
 
-# verify RAX MaaS is running after all necessary
-# playbooks have been run
+# Download the latest release of rpc-maas
+# TODO(odyssey4me):
+# Remove this once rpc-gating no longer tries
+# to run rpc-maas as its own thing and instead
+# just uses deploy.sh end-to-end. This line
+# should not be necessary as setup-maas.yml
+# below includes maas-get.yml
+run_ansible maas-get.yml
+
+# Deploy and configure RAX MaaS
 if [[ "${DEPLOY_MAAS}" == "yes" ]]; then
-  run_ansible verify-maas.yml
+    # Run the rpc-maas setup process
+    run_ansible setup-maas.yml
+
+    # verify RAX MaaS is running after all necessary
+    # playbooks have been run
+    run_ansible verify-maas.yml
+fi
+
+# To send data to the influxdb server, we need to deploy and
```

Comment:
Remove all the following lines, I am including them on https://github.com/rcbops/rpc-openstack/pull/2393/files
Alternatively, add the proper variables. You can also cherry pick my commit on top of yours.